poetry.lock

```diff
@@ -690,5 +689,5 @@ files = [
 
 [metadata]
 lock-version = "2.1"
-python-versions = "^3.9.2"
-content-hash = "06839200089536c4763132f321415b804fa5eb925b9d36b45c7fe5708219a36d"
+python-versions = "^3.10"
```

Comment:
These changes should also not be happening. Your changes shouldn't touch the poetry.lock file at all.